pkg/component/kubernetes/controllermanager/controllermanager_test.go

```diff
@@ -708,22 +614,21 @@ namespace: kube-system
 
 				Expect(kubeControllerManager.Deploy(ctx)).To(Succeed())
 
-				verifyDeployment(config, hvpaConfig, controllerWorkers, versionutils.ConstraintK8sGreaterEqual126.Check(runtimeKubernetesVersion))
+				verifyDeployment(config, isScaleDownDisabled, controllerWorkers, versionutils.ConstraintK8sGreaterEqual126.Check(runtimeKubernetesVersion))
 			},
 
-			Entry("w/o config k8s >=1.26", emptyConfig, hvpaConfigDisabled, runtimeKubernetesVersion),
-			Entry("w/o config k8s < 1.26", emptyConfig, hvpaConfigDisabled, semver.MustParse("1.25.0")),
-			Entry("with HVPA", emptyConfig, hvpaConfigEnabled, runtimeKubernetesVersion),
-			Entry("with HVPA and custom scale-down update mode", emptyConfig, hvpaConfigEnabledScaleDownOff, runtimeKubernetesVersion),
-			Entry("with non-default autoscaler config", configWithAutoscalerConfig, hvpaConfigDisabled, runtimeKubernetesVersion),
-			Entry("with feature flags", configWithFeatureFlags, hvpaConfigDisabled, runtimeKubernetesVersion),
-			Entry("with NodeCIDRMaskSize", configWithNodeCIDRMaskSize, hvpaConfigDisabled, runtimeKubernetesVersion),
-			Entry("with PodEvictionTimeout", configWithPodEvictionTimeout, hvpaConfigDisabled, runtimeKubernetesVersion),
-			Entry("with NodeMonitorGradePeriod", configWithNodeMonitorGracePeriod, hvpaConfigDisabled, runtimeKubernetesVersion),
+			Entry("w/o config k8s >=1.26", emptyConfig, false, runtimeKubernetesVersion),
+			Entry("w/o config k8s < 1.26", emptyConfig, false, semver.MustParse("1.25.0")),
+			Entry("with scale-down disabled", emptyConfig, true, runtimeKubernetesVersion),
+			Entry("with non-default autoscaler config", configWithAutoscalerConfig, false, runtimeKubernetesVersion),
+			Entry("with feature flags", configWithFeatureFlags, false, runtimeKubernetesVersion),
+			Entry("with NodeCIDRMaskSize", configWithNodeCIDRMaskSize, false, runtimeKubernetesVersion),
+			Entry("with PodEvictionTimeout", configWithPodEvictionTimeout, false, runtimeKubernetesVersion),
+			Entry("with NodeMonitorGradePeriod", configWithNodeMonitorGracePeriod, false, runtimeKubernetesVersion),
```

Comment:
Typo that was present even before
```suggestion
			Entry("with NodeMonitorGracePeriod", configWithNodeMonitorGracePeriod, false, runtimeKubernetesVersion),
```

Reply:
Done.